django_airavata/apps/workspace/static/django_airavata_workspace/js/components/experiment/ExperimentEditor.vue

```diff
@@ -163,8 +164,10 @@ export default {
   data() {
     return {
       projects: [],
+      userfiles:{},
```

Comment:
Are these used anywhere? I'm wondering if any of the changes to this file are necessary, if not, perhaps just remove your changes from this file.